### DISCUSSION TITLE
The problem with space mods. My take, & my solution idea!

### DISCUSSION THREAD

**Discussion by CreeprVictor** [terrarium-earth/Ad-Astra#443] [Ideas]
It is a big shame that all space mods we have are like this; un-original; in-unique… All of them have nothing special, heck, the moon doesn't even have creators! "Pygros" heh... those are **literally** water-downed Piglin, exact same code!
Nothing is original, not− one− feature− is special. …

But… unfortunately… We are forced to get no better…
Take this example metaphor,
_You are in a cave, you have no food, no means of obtaining more, all you have is rotten flesh. What do you do? Eat the rotten flesh, even though it is garbage, it's all you got… take what you got._
`(With the rotten flesh being current space mods (& their addons), and the cave and its food options(being current space mods & even their addons)`

Also, what is up with those Corrupt Lurinians??  Beyond un-original… like, what the heck? You're just gonna throw them scattered wandering the surface???

### **Here is my idea! (Shortened a little)**

Telescope stand, paired with spyglass lets you find the other planets and stars. Then you can go to an astronomy table to see the mapped planets and stars.

Build rocket, leave atmosphere, stay in rocket, use navigation to navigate making use of astronomy table.Go to all 8 other planets + asteroid belt & Pluto. All unique. Also go to their moons! (though, Jupiter and Saturn would be limited to 5 max)

Use warp drive to travel to other stars. Other star's planets will make use of a tweaked, balanced, etc version of the 20w14∞ April fools update w\ infinite dimensions.

And more!

**Comment by CodexAdrian** [comment]
We already have a content update planned because we fundamentally agree that the mod is derivative and boring. Its like this because the mod was originally a port of Beyond Earth which itself was fairly derivative. All the planets are being reworked to add unique content worth exploring.

Ill leave this discussion open if you want to talk about other ideas, but to address what you said:
- we already plan on removing all existing mobs from the mod
- all the planets will be remade and wont be a single dessert biome
- we dont plan on adding the planets moons right away as it would be quite alot of work to add 10+ new explorable dimensions. 
   - we plan on making every new celestial body interesting with its own unique flora and fauna, an ecosystem thats interesting. We wouldn't be able to do that if we had to do it with 20+ celestial bodies

**Reply by CreeprVictor** [reply]
Wow, nice! okay, I guess here are some more ideas if you like! (Ideas can be saved for later use of-course)


### Planets
- That Jupiter and Saturn should take place in the sky, with the land their dense clouds, falling down, you would die in the "void", as down there, the pressure is too great since those 2 planets are fully made up of gas.
- Uranus and Neptune have a slushy-like surface, since those 2 planets have no, "surface" per say, but rather an ice-slushy-like mantle.
- You can land on Saturn's rings, (Jupiter's and Uranus's would be lightly visible, but can't be landed on.
- Different planets have different day lengths
- You can land at the asteroid belt
- Venus has 2 parts, the land, and the sky. The land you catch on fire (like normal) and the sky, has breathable air, and uses the same concept as Jupiter and Saturn, with dense clouds you can stand on.
- Mercury's rock / sand has a secret color map that resembles those beautiful photos of Mercury under that filter or whatever, in which, spectating any mob from Mercury shows this colormap!
- You could go to the sun, but you just die, just for fun lol.

- Jupiter's moons: Europa, Io, Ganymede, Calisto
- Saturn's moon(s): Titan & ,not a moon, but the rings

### Biome Ideas
- Mercury Wasteland
- Mercury, ehh, Idk much

- Venusian Desert
- Venusian Dunes
- Venusian Deltas
- Venusian Skies

- Lunar Wastelands
- Lunar Large Crater
- Lunar Poles

- Marian Plains
- Marian  Dry River
- Marian Dry Lake
- Marian Poles

- Asteroid Belt Gap
- Rocky Asteroids
- Icy Asteroids

- Jupiter Skies
- Jupiter Storms

- Saturn,

- Uranus,

- Neptune,
- Don't forget about Neptune's big blue spot

- Pluto,





### Astromony
You first make a Level1 Astronomy Table, now you unlock the ability to look at the moon (Luna) via a telescope (spyglass on a tripod)..
The moon now has been saved to your findings, in which you can now see whenever you go to a level 1+ Astronomy table.
Tier 1 rocket, go to moon,

Then with a level 2 Astronomy table you can map out the other planets from Sol(sun) to Uranus (and the asteroid belt is findable now)
Then you'd need to rank up the rocket for further,

Then with a level 3 Astronomy table you can find Neptune, Pluto, and the stars that have the other new planets you want to add!

To find the planets at other stars, you would need to map the sun, then set the telescope to focus on that star, leave it, and it will eventually map a random planet from that star! From there, you can find the other planets from on that one!



### Procedurally Generated Planets (& Stars)
This one is more complex, If you would like to add this at some point, reach out to me, I've got the idea planned out quite a bit!

**Reply by CreeprVictor** [reply]
When should we expect the next Ad Astra Update to revamp the planets?
